cmd/tekton-mcp-server/main.go

```diff
@@ -55,18 +63,23 @@ func main() {
 	resources.Add(ctx, s)
 
 	slog.Info("Starting the server.")
-	// Start the stdio server
-	stdioServer := server.NewStdioServer(s)
-	// Start listening for messages
+
 	errC := make(chan error, 1)
-	go func() {
-		in, out := io.Reader(os.Stdin), io.Writer(os.Stdout)
 
-		errC <- stdioServer.Listen(ctx, in, out)
-	}()
+	if transport == "sse" {
+		sseServer := server.NewSSEServer(s, server.WithSSEContextFunc(func(_ context.Context, r *http.Request) context.Context { return ctx }))
+		go func() {
+			errC <- sseServer.Start(sseAddr)
+		}()
 
-	// Output tekton-mcp string
-	_, _ = fmt.Fprintf(os.Stderr, "Tekton MCP Server running on stdio\n")
+		slog.Info("Tekton MCP Server is listening at " + sseAddr)
+	} else {
```

Comment:
With this, I can give a flag `--transport=toto` and it will work (and start stdio). We should probably return an error in case of an invalid transport flag.